test/cluster.sh

```diff
@@ -48,8 +48,14 @@ function install_knative_serving() {
   kubectl apply -f "${ISTIO_YAML}" || return 1
 
   echo ">> Bringing up Serving"
-  # TODO(#2122): Use RELEASE_YAML once we have monitoring e2e.
-  kubectl apply -f "${RELEASE_NO_MON_YAML}" || return 1
```

Comment:
This will break the upgrade tests. Instead of declaring yet another env var `USE_MONITORING` and have yet another different flow, just override `RELEASE_YAML_OVERRIDE` to `$RELEASE_YAML`. This will also automatically gracefully handle the uninstall, which you're not currently doing.